src/extension/VSProject.cs

```diff
@@ -348,9 +348,13 @@ private void LoadMSBuildProject()
             MsBuildDocument = _doc;
 
             XmlElement assemblyNameElement = (XmlElement)_doc.SelectSingleNode("/msbuild:Project/msbuild:PropertyGroup/msbuild:AssemblyName", namespaceManager);
+            if (assemblyNameElement == null)
+                throw new ApplicationException("Missing required element AssemblyName");
             string assemblyName = assemblyNameElement.InnerText;
 
             XmlElement outputTypeElement = (XmlElement)_doc.SelectSingleNode("/msbuild:Project/msbuild:PropertyGroup/msbuild:OutputType", namespaceManager);
+            if (outputTypeElement == null)
+                throw new ApplicationException("Missing required element OutputType");
```

Comment:
And the exception is a little ugly - a raw stack trace in the console. I haven't tried it in Visual Studio yet, but I'll update here.